### PR TITLE
HelpSource: Fix typos in Literals help file

### PR DESCRIPTION
**Pull request by hectorgonzalezo**
## Purpose and Motivation
Fix a couple of typos in the "Literals" help file

## Types of changes

- Documentation

## To-do list

- [x] Code is tested
- [x] All tests are passing
- [x] Updated documentation
- [x] This PR is ready for review

### CODE REVIEW

File: HelpSource/Reference/Literals.schelp [hunk 1]
```diff
@@ -348,7 +348,7 @@ code::
 
 The outer function includes only the selector code::value::. The other selectors -- code::>::, code::*::, code::-:: -- belong to the inner function definition and don't affect the outer function's number of selectors.
 
-So, one possible easy way to work around the limitation is to break up a large block of code into several functions that are value'd successively:
+So, one possible easy way to work around the limitation is to break up a large block of code into several functions that are valued successively:
```
Comment (dyfer) [inline HelpSource/Reference/Literals.schelp:351]: I believe this was intended as `value'd`, meaning that several functions are being evaluated with the `.value` method.

**Review by dyfer — CHANGES_REQUESTED** [review]
Thanks for the corrections! 
I believe one of them is not needed, please see my review.

File: HelpSource/Reference/Literals.schelp (hunk 1 above)
Comment (hectorgonzalezo) [reply HelpSource/Reference/Literals.schelp:351]: Oh, I see. Then `.value`'d would definitely make more sense. Thanks!

**Comment by hectorgonzalezo** [comment]
Ok, left out the value'd part entirely. Let me know if I'm missing anything, as this is my first PR for SuperCollider.

By the way, that piece on your webpage home is awesome, congrats! I do a lot of Interactive Computer Music as well, hope to have a chance to chat some day.

**Review by dyfer — APPROVED** [review]

**Pull request merged**